Power Up Proto/src/org/usfirst/frc/team5493/robot/subsystems/OneClimbyBoi.java

```diff
@@ -0,0 +1,70 @@
+package org.usfirst.frc.team5493.robot.subsystems;
+
+import org.usfirst.frc.team5493.robot.Robot;
+import org.usfirst.frc.team5493.robot.RobotMap;
+
+import com.ctre.phoenix.motorcontrol.can.WPI_TalonSRX;
+
+import edu.wpi.first.wpilibj.CounterBase.EncodingType;
+import edu.wpi.first.wpilibj.Encoder;
+import edu.wpi.first.wpilibj.command.Subsystem;
+
+/**
+ *
+ */
+public class OneClimbyBoi extends Subsystem {
+
+	public WPI_TalonSRX climbMotor;
+	public Object climbDown;
+	
+	public static final double kDistancePerRevolution = 18.84;
+	public static final double kPulsesPerRevolution = 1024;
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Rename this constant name to match the regular expression '^[A-Z][A-Z0-9]*(_[A-Z0-9]+)*$'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS00115)